Makefile

```
@@ -3,6 +3,7 @@
 GO=go
 LDFLAGS?=-s -w
 TESTFILE=_testok
+MOUNT_PATH= /local
```

[optional] not sure if it gets trimmed already in some other stages, probably when mounting.

```suggestion
MOUNT_PATH=/local
```